crates/solvers/src/boundary/baseline.rs

```diff
@@ -13,7 +13,7 @@ use {
 
 pub struct Solver<'a> {
     base_tokens: BaseTokens,
-    amms: HashMap<TokenPair, Vec<Amm>>,
+    amms: HashMap<TokenPair, Vec<OnchainLiquidity>>,
```

Comment:
What's the difference between amms and liquidity? Does it make sense to combine the two somehow (maybe in a separate PR)? Also `amms` should probably be renamed.

Reply:
Renamed and created an issue https://github.com/cowprotocol/services/issues/2332